test/metabase/models/query_field_test.clj

```diff
@@ -127,3 +129,24 @@
           ;; subset since it also includes the PKs/FKs
           (is (set/subset? #{total-qf tax-qf}
                            (t2/select-fn-set qf->map :model/QueryField :card_id card-id :direct_reference true))))))))
+
+(deftest parse-mbql-test
+  (testing "Parsing MBQL query returns correct used fields"
+    (mt/with-temp [Card c1 {:dataset_query (mt/mbql-query venues
```

Comment:
```suggestion
    (mt/with-temp [:model/Card c1 {:dataset_query (mt/mbql-query venues
```

if you do this then you don't need to import `Card`. And it's actually the recommended way going forward as we want to remove `Card` symbol one day.

Reply:
Yeah but this kinda bites you once in a while in tests exactly. You restart Metabase, did not import `metabase.models` in your tests, eval the file, run the tests — and errors are going to be weird.

So I actually added a symbol for `CacheConfig` just to prevent this happening.